test_runner/fixtures/zenith_fixtures.py

```diff
@@ -226,6 +226,11 @@ def tenant_create(self, tenant_id: uuid.UUID):
         res.raise_for_status()
         return res.json()
 
+    def get_metrics(self) -> str:
+        res = self.get(f"http://localhost:{self.port}/metrics")
+        res.raise_for_status()
+        return res.text
```

Comment:
It's probably ok to return raw string, since we only use one metric. However, I'd add a small TODO comment regarding parsing the whole string into a convenient structure (possibly dropping dull metrics).

Reply:
added a TODO comment about that